<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConTransfDialog</class>
 <widget class="QDialog" name="ConTransfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>176</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>423</width>
    <height>176</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>176</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit_from">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="placeholderText">
      <string>Source account number(leave empty, if you want to fill account)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_to">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="placeholderText">
      <string>Destination account number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_amount">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>129</red>
           <green>130</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="placeholderText">
      <string>Amount of money</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_buttons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_transfer">
        <property name="text">
         <string>Transfer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
